src/app/SDLApp.cpp

```diff
@@ -148,6 +152,8 @@ void SDLApp::render(void)
 
     ImGui::End();
 
+    _serialConfigGui.render();
```

Comment:
You can remove the old dummy window here as its purpose was having imgui running